react/features/settings/components/web/SettingsDialog.tsx

```diff
@@ -297,6 +300,25 @@ function _mapStateToProps(state: IReduxState, ownProps: any) {
         });
     }
 
+    if (showSoundsSettings || showNotificationsSettings) {
+        tabs.push({
+            name: SETTINGS_TABS.NOTIFICATIONS,
+            component: NotificationsTab,
+            labelKey: 'settings.notifications',
+            propsUpdateFunction: (tabState: any, newProps: any) => {
+                return {
+                    ...newProps,
+                    enabledNotifications: tabState?.enabledNotifications || {}
+                };
+            },
+            props: getNotificationsTabProps(state),
```

Comment:
Since we already have the `showSoundsSettings` here, maybe pass it directly from here to props rather than computing it again in the component's `mapStateToProps`, that way we can have one place to remove it from, once we remove the support for the interface config key.